addons/remember-collapsed-categories/userscript.js

```diff
@@ -0,0 +1,54 @@
+export default async function ({ console }) {
+  const categories = document.querySelectorAll('[id^="category_body_"]');
```

Comment:
Using document.querySelectorAll instead of waitForElement could break because you're running this with `"runAtComplete" false`.
Either run this at complete, or use a while(true) pattern with waitForElement like other addons.

Reply:
Done... I think.